util/git.go

```diff
@@ -14,3 +15,25 @@ func UpdateGitRepo(ctx context.Context, gitpath string) {
 	Debugf(ctx, "%s: run %s\n", gitpath, cmd)
 	CheckCmd(cmd.CombinedOutput())
 }
+
+func IsPathIgnoredByGit(ctx context.Context, gitpath string, path string) bool {
+	// We don't know if "path" is a file or a directory, so let's try with and without /
+	return isPathIgnoredByGit(ctx, gitpath, path) || isPathIgnoredByGit(ctx, gitpath, path+"/")
+}
+
+func isPathIgnoredByGit(ctx context.Context, gitpath string, path string) bool {
+	// We need a relative path, so let's remove "gitpath"
+	path = strings.TrimPrefix(path, gitpath)
+	// In case "path" is a absolute path, we need to remove "/" afterwards to get a relative path
+	path = strings.TrimPrefix(path, "/")
+	args := []string{"check-ignore", "--quiet", "--no-index", path}
+
+	cmd := exec.Command("git", args...)
+	cmd.Dir = gitpath
+	Debugf(ctx, "%s: run %s\n", gitpath, cmd)
+
+	if err := cmd.Run(); err != nil {
```

Comment:
maybe we can change this to just `return cmd.Run() == nil`? if that's confusing I'm fine with how it is currently 😄